src/lib/annotations/Annotator.js

```diff
@@ -205,6 +204,17 @@ class Annotator extends EventEmitter {
     }
 
     /**
+     * Returns true if the annotator has an annotation type enabled
+     *
+     * @param {string} type - Annotation type to check
+     * @return {boolean} Whether or not the annotation type is enabled
+     */
+    isTypeEnabled(type) {
```

Comment:
Yeah I initially added it as a check before isModeAnnotatable was there. Gonna remove this